include/seqan3/core/algorithm/detail/blocking_algorithm_executor.hpp

```diff
@@ -293,14 +265,14 @@ class blocking_algorithm_executor
      * Clears all buckets and sets the bucket iterator to the first iterator, such that the allocated memory for each
      * bucket can be reused between invocations of seqan3::detail::blocking_algorithm_executor::underflow.
      */
-    void reset_buckets()
+    void reset_buffer()
     {
         // Clear all buckets
-        for (auto & bucket : bucket_vector)
+        for (auto & bucket : buffer)
             bucket.clear();
 
         // Reset the iterator over the buckets.
-        bucket_iterator = bucket_vector.begin();
+        buffer_it = buffer.begin();
     }
 
     /*!\brief Finds the first non-empty bucket starting from the current bucket iterator.
```

Comment:
should be changed to:
Finds the first non-empty bucket starting from the current `buffer_it`. ?